.eslintrc.json

```diff
@@ -20,7 +20,8 @@
         "plugin:react/recommended",
         "plugin:react-hooks/recommended",
         "plugin:security/recommended",
-        "plugin:import/typescript"
+        "plugin:import/typescript",
+        "prettier"
```

Comment:
I'll double check to see what the behavior is exactly.  We made need to add an explicit format check/step to the CI process.

Reply:
The prettier docs suggest a separation here to have linting rules not look at formatting, but to still check for code errors and just have prettier handle formatting/presentation.  https://prettier.io/docs/en/comparison.html

Reply:
I think having style be enforced will be important to avoid the code churn/irrelevant changes we're facing today (bonus points if the workflow can just auto commit a fix instead of failing). I don't have a strong preference about whether it's done as a separate step or as part of ES Lint.

Reply:
I'll open a second PR to make the pipeline changes.  Will definitely auto-fix; however, with tests that's likely to break something (@ts-ignore) statements need to be on the line above the offending bit of code.